test/make.sh

```
@@ -97,12 +102,35 @@ function generate_docs() {
   rm -f "$TMPFILE"
 }
 
+function check_generate() {
+  TMPDIR=$(mktemp -d)
+  git worktree add --detach "$TMPDIR" >/dev/null
+  cd "$TMPDIR" || exit 1
+
+  generate >/dev/null
```

Will this silently swallow errors?